deploy/charts/rook-ceph-cluster/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,32 @@
+{{/*
```

Comment:
This file doesn't exist in master. Is it intentional, or maybe it isn't needed anymore? I don't recall.